src/makeCalculator.js

```diff
@@ -37,7 +37,44 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    Methods: [`add`, `subtract`, `multiply`, `divide`, `reset`, `operate`],
```

Comment:
why do you need this property?

Reply:
when I was solving the task, I thought that I have return possible operations 😕